k8s/src/test/scala/io/buoyant/k8s/EndpointsNamerTest.scala

```diff
@@ -1165,27 +1165,31 @@ class EndpointsNamerTest extends FunSuite with Awaits {
     val timer = new MockTimer
     val namer = new MultiNsNamer(Path.read("/test"), None, api.withNamespace, Stream.continually(1.millis))(timer)
 
-    @volatile var state: Activity.State[NameTree[Name]] = Activity.Pending
+    @volatile var state: Set[Address] = Set.empty
 
     val activity = namer.lookup(Path.read("/srv/http/sessions"))
 
-    eventually {
-      val addrs = await(activity.toFuture.map {
-        case NameTree.Leaf(Name.Bound(vaddr)) => vaddr.sample() match {
```

Comment:
The problem with this test is that we only sample the vaddr, never observe it long enough for it to reach its final value.  Previously this was fine because observing the outer Var drove updates to the vaddr.  With our change to stabilizeExistence, we need to observe the vaddr itself if we want it to receive updates.